services/src/main/java/io/grpc/protobuf/services/ProtoReflectionService.java

```diff
@@ -77,8 +77,7 @@
   private ProtoReflectionService() {}
 
   /**
-   * Creates a instance of {@link ProtoReflectionService}. Intended usage is one instance per
```

Comment:
Doh. I missed this before. Of course it was already limited to one server. Sorry for asking for the WeakHashMap, although it is nicer that way. I guess previously we'd have just returned wrong data? Yeah, using the WeakHashMap is much nicer for users.